### PR TITLE
Add PYRODRONEF4 unified target

### PR DESCRIPTION
**Pull request by bnn1044**
Add PYRODRONEF4 unified target

### CODE REVIEW

File: configs/default/PYRO-PYRODRONEF4.config [hunk 1]
```diff
@@ -0,0 +1,79 @@
+# Betaflight / STM32F405 (S405) 4.1.1 Nov 15 2019 / 12:54:53 (1e5e3d369) MSP API: 1.42
+# board_name: PYRODRONEF4   custom defaults: NO
```
Comment (mikeller) [inline configs/default/PYRO-PYRODRONEF4.config]: This line should not be in here.

**Comment by bnn1044** [comment]
@mikeller  I have changed the Manufacturer ID.

**Comment by mikeller** [comment]
The [requirements](https://github.com/betaflight/betaflight/blob/master/docs/TargetMaintenance/CreatingAUnifiedTarget.md#7-create-a-unified-target-configuration-file-for-your-board) are for the name to be `<manufacturer_id>-<board_name>.config`.

**Comment by mikeller** [comment]
There's now one too many `PYRODRONEF4` in `PYDR-PYRODRONEF4PYRODRONEF4.config`.

**Pull request merged**